src/core/eme/utils/init_data_store.ts

```diff
@@ -38,6 +38,8 @@ export default class InitDataStore<T> {
 
   /**
    * Returns all stored value, in the order in which they have been stored.
+   * Note: it is possible to move a value to the end of this array by calling
+   * the `getAndRefresh` method.
```

Comment:
getAndReuse*

Reply:
Haha!
Yep that was an old name I finally considered less adapted.

Reply:
Done